app_server.py

```diff
@@ -270,7 +270,7 @@ def work_lookup(self, annotator, route_name='lookup', **process_urn_kwargs):
         urns = flask.request.args.getlist('urn')
 
         this_url = cdn_url_for(route_name, _external=True, urn=urns)
-        response = self.process_urns(urns)
+        response = self.process_urns(urns, **process_urn_kwargs)
```

Comment:
Without this, collection information obtained from the URL is not propagated from the metadata wrangler to process_urns.